generated_code/aderdg.py

```diff
@@ -40,6 +40,7 @@
 
 import numpy as np
 from abc import ABC, abstractmethod
+from common import *
```

Comment:
We should try and avoid from common import *, especially if we are only importing one function. Better to stay explicit when importing something into the global space.

Reply:
Same of course for other instances of this line.